api/rs/slint/Cargo.toml

```diff
@@ -134,7 +134,8 @@ pin-weak = { version = "1.1", default-features = false }
 num-traits = { version = "0.2", default-features = false }
 
 log = { version = "0.4.17", optional = true }
-
+serde_json = { version = "1.0.96", optional = true } 
+serde = { version = "1.0.163", features = ["derive"], optional = true }
```

Comment:
you should also make a "serde" feature that enables "dep:serde" and also the serde feature in i-slint-core